### ISSUE TITLE
Implementation of Google Analytics

### ISSUE THREAD

**Issue by domingoruiz** [rocky-linux/rockylinux.org#127]
Hello, since we are implementing so many translations to reach many people, I think it would be a good idea to implement google analytics or some alternative to know how many people we are reaching and how they interact with the site. For this we would have to create a google account and link it to the site. In addition to this, we would have to prepare a banner that warns of the use of cookies.

What do you think?

Thanks

**Comment by FoggyMtnDrifter** [comment]
I don't think Google Analytics is a good thing for the project because of privacy concerns, but there are alternatives that are privacy oriented out there that other distributions use like [Plausible](https://github.com/plausible/analytics).

**Comment by plaimbock** [comment]
How about using https://matomo.org/ which has built-in GDPR functionality?

**Comment by domingoruiz** [comment]
Yes, I think Google Analytics is not the best option, but I think you have understood the idea that we should implement an alternative to at least know how the interaction with the site is being.

**Comment by hbjydev** [comment]
Bear in mind, TRLF has a Google Workspaces setup, so creating a Google account to link to it wouldn't be a concern for us.
In theory nor would the privacy as long as we set it up correctly.

**Comment by domingoruiz** [comment]
I also don't think anything happens to use Google Analytics as long as everything is configured correctly, but hey, that's already a team decision. What is clear that if some form of follow-up should be implemented and also some way of making that information accessible to everyone on the team.

**Comment by FoggyMtnDrifter** [comment]
I guess the biggest concern for me about Google Analytics is that more and more browsers are implementing popups to tell the user if there are trackers on the website and it might set off a red flag to someone if they see Google Analytics, regardless of how we set it up.

Plausible, the alternative I put above, doesn't use cookies, nor store IP addresses.

**Comment by FoggyMtnDrifter** [comment]
What's also neat about Plausible is that you can setup a public analytics page for everyone to view, no need to be logged in. It would increase transparency and give insights to visitors about who visit the website.

**Comment by domingoruiz** [comment]
I like all that you are commenting on Plausible. I think its implementation would be worth studying.

**Comment by FoggyMtnDrifter** [comment]
Don't get me wrong, I'm not completely opposed to Google Analytics, I'm just concerned that it may put off some users if they see we're using it on the site. If we do choose to use Google Analytics, I think we should make a pretty clear privacy policy that outlines what it's being used for.

**Comment by tejclark** [comment]
+1 for Matomo, great analytics platform and good alternative to GA. They seem to have non-profit pricing for their cloud offering if Rocky don't want to roll their own server.

Don't forget if Rocky are adding tracking there'll need to add in some form of Cookie banner to cover EU cookie law. I've used OneTrust before which worked well for that if you just want something "out of the box".

**Comment by danieltharp** [comment]
> 
> 
> Don't get me wrong, I'm not completely opposed to Google Analytics, I'm just concerned that it may put off some users if they see we're using it on the site. If we do choose to use Google Analytics, I think we should make a pretty clear privacy policy that outlines what it's being used for.

I would generally agree that using Google Analytics and seeing the notification as such that the script was blocked could affect perception around the site. Since we're looking at self-hosted tools for static site generation and hosting I think we should ensure we can attach an analytics system with minimal work. Plausible does look good in that regard and looks like it has the core features that I think are worth keeping an eye on, high-traffic warnings, 404 reports, and GDPR compliance.
